test_scripts/API/AddCommand/ATF_AddCommand_UNSUP_RES_to_UI_AddCmd_WARNINGS_to_VR_AddCmd.lua

```diff
@@ -0,0 +1,134 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one component of RPC
+--
+-- Description:
+-- test is intended to check that SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- in this particular test it is checked case when VR.AddCommand gets Warnings and UI.AddCommand gets UNSUPPORTED_RESOURCE from HMI
+-- 1. Used preconditions: App is activated and registered SUCCESSFULLY
+-- 2. Performed steps: 
+-- MOB -> SDL: sends AddCommand
+-- SDL -> HMI: resends VR.AddCommand and UI.AddCommand
```

Comment:
@istoimenova , Please see updated in commit e3c195283c2acd34494b065de5dde8270b56f351